cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -31,6 +31,48 @@ Cask Data Application Platform Release Notes
    :depth: 2
 
 
+`Release 4.3.3 <http://docs.cask.co/cdap/4.3.3/index.html>`__
+=============================================================
+Improvements
+------------
+
+- :cask-issue:`CDAP-12942` - GroupBy aggregator plugin fields are now macro enabled.
+
+- :cask-issue:`CDAP-12963` - Allow CDAP user programs to talk to Kerberos enabled HiveServer2 in the cluster without using a keytab.
+
+- :cask-issue:`CDAP-12974` - Removed concurrent upgrades of HBase coprocessors since it could lead to regions getting stuck in transit.
+
+Bug Fixes
+---------
+
+- :cask-issue:`CDAP-7052` - Fixed a bug that prevented MapReduce AM logs from YARN to show the right URI.
+
+- :cask-issue:`CDAP-7644` - Added CLI command to fetch service logs.
+
+- :cask-issue:`CDAP-12774` - Increased the dataset changeset size and limit to integer max by default.
+
+- :cask-issue:`CDAP-12900` - Fixed bug where macro for output schema of a node was not saved when the user closed the node properties modal.
+
+- :cask-issue:`CDAP-12930` - Fixed a bug where explore queries would fail against paths in hdfs encryption zones, for certain hadoop distributions.
+
+- :cask-issue:`CDAP-12945` - Fixed bug where old connection was not removed from the pipeline config when we moved that connection's pointer to another node.
```

Comment:
"Fixed a bug where the old connection is not removed from the pipeline config when you move the connection's pointer to another node."

Reply:
fixed.